speed-dreams/source-2.2.3/data/blackbox/run.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
```

Comment:
A commit mentions that this file is deleted, but it is still here, why is that

Reply:
It is not; you probably started your review before I made that commit.